### PR TITLE
Add steps to remove contaminated files from the DB and S3 buckets

### PR DESCRIPTION
**Pull request by vimleshtna**

### CODE REVIEW

File: manual/Remove-contaminated-files-from-tdr.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+## SSH tunneling to connect bastion from your local machine - (required to delete from intg/staging database)
+
+1. Install the latest aws cli on your machine.
+2. Create an AWS CLI profile
+   ```
+   aws configure sso
+   aws sso login --profile tdr-integration ( change to the profile you created)
+   ```
+3. Add your ssh key to the bastion
```
Comment (thanhz) [inline manual/Remove-contaminated-files-from-tdr.md:13]: Can you add a note about how we can alternatively get the public key from our computer and add it to the run bastion github action public key field? 

I think it was something like
cd to the .ssh folder and
cat id_ed25519.pub?

File: manual/Remove-contaminated-files-from-tdr.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    - sudo su ec2-user
+    - cd .ssh
+    - nano authorised_key
+4. On your machine add a line to your .ssh config file
```
Comment (thanhz) [inline manual/Remove-contaminated-files-from-tdr.md:17]: Can you mention explicitly the directory for the steps i.e. I think I remember we modified files in the following directory
/etc/ssh/ssh_config
/etc/hosts

**Review by thanhz — COMMENTED** [review]
Looks good, just a few comments

**Review by thanhz — COMMENTED** [review]
Thanks, just a few more minor comments

File: manual/Remove-contaminated-files-from-tdr.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+    ``` 
+   Or if you know the file id of the contaminated file then first find the checksum and then use it to find all the affected files.</br>
+   **Once you get the results, save them in a notepad because you need these details to delete files from the S3 buckets**
+
+3. Use the following sequence to delete all the references of affected consignments from the database:
+
+   | Table               | Query to delete                                                                                                                                                                                           |
+   |---------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+   | AVMetadata          | `delete from "AVMetadata" where "FileId" in (select "FileId" from "File" where "ConsignmentId" in ('CONSIGNMENT IDs'));`                                                                                  |
```
[the remark this answers is not in the document]
Comment (vimleshtna) [reply manual/Remove-contaminated-files-from-tdr.md:58]: I have added a note for this. Thanks

**Review by thanhz — APPROVED** [review]

**Pull request merged**